web/src/components/FilingsCalendar.test.tsx

```diff
@@ -648,5 +666,29 @@ describe("<FilingsCalendar />", () => {
       renderFilingsCalendar(operateReferences, userData);
       expect(screen.queryByText(Config.dashboardDefaults.calendarGridViewButton)).not.toBeInTheDocument();
     });
+
+    it("displays the calendar default view as list view when the taxFilingData state is 'PENDING'", () => {
+      userData = {
+        ...userData,
+        preferences: { ...userData.preferences, isCalendarFullView: false },
+        taxFilingData: { ...userData.taxFilingData, state: "PENDING" },
+      };
+
+      renderFilingsCalendar(operateReferences, userData);
+
+      expect(screen.getByTestId("filings-calendar-as-list")).toBeInTheDocument();
+    });
+
+    it("doesn't display the toggle button when the taxFilingData state is 'PENDING'", () => {
+      userData = {
+        ...userData,
+        preferences: { ...userData.preferences, isCalendarFullView: false },
+        taxFilingData: { ...userData.taxFilingData, state: "PENDING" },
+      };
+
+      renderFilingsCalendar(operateReferences, userData);
+
+      expect(screen.getByText(Config.dashboardDefaults.calendarGridViewButton)).toBeInTheDocument();
+    });
   });
```

Comment:
Why has this test file changed if none of the code for the FilingsCalendar is changed? I think that all these changes can be removed, right?